src/config/config.ts

```diff
@@ -283,6 +283,13 @@ const config = convict({
       format: "required-string",
       default: "",
     },
+    siteCloneFormKey: {
```

Comment:
This needs to be added to `.platform/hooks/predeploy/06_fetch_ssm_parameters.sh`

Reply:
thanks for the catch!